runtime/doc/provider.txt

```diff
@@ -238,6 +238,9 @@ The "copy" function stores a list of lines and the register type. The "paste"
 function returns the clipboard as a `[lines, regtype]` list, where `lines` is
 a list of lines and `regtype` is a register type conforming to |setreg()|.
 
+Set |g:clipboard| to |v:false| to explicitly disable all clipboards not listed
```

Comment:
What do you think about putting ``|g:clipboard| (unless unset or `false`)`` on line 188 instead?